content/millennium/dstu2/infrastructure/binary.md

```diff
@@ -10,12 +10,13 @@ title: Binary | DSTU 2 API
 ## Overview
 
 The Binary resource can contain any clinical content such as text, images, and PDFs.  This resource is currently limited to Continuity of Care Documents (CCD), clinical documents, and diagnostic reports.
+_The list of clinical documents and diagnostic reports is subject to change, as the list of clinical events that may qualify can be customized by clients_
 
-It is recommended to request all Binary resources only after obtaining a link to the resource via references from DiagnosticReport or DocumentReference. It is not recommended to start a workflow in the Binary resource.
+It is recommended to request Binary resources only after obtaining a link to the resource via references from [DiagnosticReport](http://fhir.cerner.com/millennium/dstu2/diagnostic/diagnostic-report/) or [DocumentReference](http://fhir.cerner.com/millennium/dstu2/infrastructure/document-reference/). It is not recommended to start a workflow in the Binary resource.
 
-The consumer must populate the Accept header with either application/json+fhir or the format returned in the attachment.contentType of the referring resource.  If the Accept header is application/json+fhir, a FHIR Binary resource is returned with the raw data populated in the content. Otherwise, the raw data will be returned (not contained within a FHIR resource). For more information see [the Binary Documentation](http://hl7.org/fhir/dstu2/binary.html#rest).
+The consumer must populate the `Accept` header with either `application/json+fhir` or the format returned in the `attachment.contentType` of the referring resource.  If the `Accept` header is `application/json+fhir`, a FHIR Binary resource is returned with the raw data populated in the content. Otherwise, the raw data will be returned (not contained within a FHIR resource). For more information see the HL7<sup>®</sup> FHIR<sup>®</sup> [Binary Documentation](http://hl7.org/fhir/dstu2/binary.html#rest).
 
-NOTE: To retrieve documents with application/pdf contentType, the client must have installed Cerner’s Clinical Reporting (XR) solution. For additional information about Cerner’s Clinical Reporting (XR) solution see [Clinical Reporting (XR) Reference Pages](https://wiki.ucern.com/display/reference/Clinical+Reporting+XR+Reference+Pages).
+NOTE: To retrieve documents with `application/pdf` contentType, the client must have installed Cerner’s Clinical Reporting (XR) solution. For additional information about Cerner’s Clinical Reporting (XR) solution see [Clinical Reporting (XR) Reference Pages](https://wiki.ucern.com/display/reference/Clinical+Reporting+XR+Reference+Pages).
```

Comment:
not sure on the updated naming, but I know we've had some communication already. is something we should note for some point